lib/metasploit/framework/credential_collection.rb

```diff
@@ -88,18 +88,10 @@ def prepend_cred(cred)
     # @yieldparam credential [Metasploit::Framework::Credential]
     # @return [void]
     def each_filtered
-      if password_spray
```

Comment:
The class hierarchy here is:

```
class PrivateCredentialCollection
  # ...
end

class CredentialCollection < PrivateCredentialCollection
  # ...
end
```

The pull request (class PrivateCredentialCollection) added password spray support to `PrivateCredentialCollection` - but it should have been added to `CredentialCollection`, as `PrivateCredentialCollection` knows nothing about usernames